plugin.php

```diff
@@ -19,11 +19,13 @@
 
 // Constants.
 define( 'MKDO_LPFC_ROOT', __FILE__ );
+define( 'MKDO_LPFC_NAME', 'Link Picker for CMB2' );
+define( 'MKDO_LPFC_PREFIX', 'mkdo_lpfc' );
 
-// Load Classes.
-require_once 'php/class-controller-main.php';
-require_once 'php/class-controller-assets.php';
-require_once 'php/class-render-meta-box.php';
+// Auto Load Classes from the php/ folder.
+foreach ( glob( plugin_dir_path( __FILE__ ) . 'php/class-*.php' ) as $filename ) {
```

Comment:
Good thinking 👍. I'd also like to move the functions out of the main controller into the main plugin file, but this can wait for another day.